armi/bookkeeping/db/tests/test_comparedb3.py

```diff
@@ -180,7 +180,7 @@ def test_compareDatabaseSim(self):
                 dbs[1]._fullPath,
                 timestepCompare=[(0, 0), (0, 1)],
             )
-        self.assertEqual(len(diffs.diffs), 504)
```

Comment:
this change is fine, but it feels like a weak test 🤷

Reply:
Agreed.

Actually, you know what?  It's Friday, I'm just going to improve that test.

There, now it is a LOT less fragile. This is kind of "doing two things in one PR", but this test just breaks all the time and I hate it.